### PR TITLE
Feature/playa interactions

### PR DESCRIPTION
**Pull request by thecadams**

### CODE REVIEW

File: src/components/templates/Camp/components/Map.scss [hunk 1]
```diff
@@ -44,7 +44,6 @@ $border-radius: 28px;
       display: block;
     }
     .playa-venue-img {
-      box-shadow: 0 8px 20px 0 rgba(0, 0, 0, 0.2);
```
Comment (thecadams) [inline src/components/templates/Camp/components/Map.scss:47]: whoops these are unrelated fixes for camp venue where it had some unnneeded shadows.

File: src/components/templates/Camp/components/Map.scss [hunk 2]
```diff
@@ -65,7 +64,6 @@ $border-radius: 28px;
     position: relative;
     width: 100%;
     height: 100%;
-    box-shadow: 0 4px 6px 0 rgba(0, 0, 0, 0.2);
```
Comment (thecadams) [inline src/components/templates/Camp/components/Map.scss:68]: whoops these are unrelated fixes for camp venue where it had some unnneeded shadows.

File: src/components/templates/Playa/AvatarLayer.tsx [hunk 3]
```diff
@@ -125,43 +127,69 @@ const AvatarLayer: React.FunctionComponent<PropsType> = ({
...
+        }),
+    [partygoers, user, userStateMap]
+  );
+
+  return useMemo(
+    () => (
+      <>
+        {myAvatar}
+        {avatars}
+        <Overlay target={hoveredRef.current} show={hovered}>
```
Comment (maxb94) [inline src/components/templates/Playa/AvatarLayer.tsx:171]: could just use hoveredUser and show={!!hoveredUser}. Will need to `setHoveredUser(null)` on mousOut

File: src/components/templates/Playa/Playa.tsx [hunk 4]
```diff
@@ -250,9 +259,7 @@ const Playa = () => {
     if (!venuePlacement) {
       return;
     }
-    return Math.sqrt(
-      Math.pow(venuePlacement.x - x, 2) + Math.pow(venuePlacement.y - y, 2)
-    );
+    return Math.hypot(venuePlacement.x - x, venuePlacement.y - y);
```
Comment (maxb94) [inline src/components/templates/Playa/Playa.tsx:262]: nice

File: src/components/templates/Playa/AvatarLayer.tsx (hunk 3 above)
Comment (thecadams) [reply src/components/templates/Playa/AvatarLayer.tsx:171]: I did that, but unsetting the hovered user makes the fading-out overlay lose its user and such, it looks super janky. So I re-added the boolean

**Review by maxb94 — APPROVED** [review]
